language-support/ts/daml-ledger/index.ts

```diff
@@ -169,6 +169,7 @@ function encodeQuery<T extends object, K, I extends string>(template: Template<T
 type LedgerResponse = {
   status: number;
   result: unknown;
+  warnings: unknown | undefined;
```

Comment:
I know it makes sense here but `unknown | undefined` still looks very weird :slightly_smiling_face:

Reply:
I guess it would be clearer if I could write:
```
warnings?: unknown
```
but for some reason our TS configuration does not allow that syntax in type declarations.